code-samples/eventing/bookstore-sample-app/frontend/client/components/CommentList.js

```diff
@@ -0,0 +1,12 @@
+import CommentDisplay from './CommentDisplay';
+const CommentList = () => {
+	const comment = {
+		avatar: '/images/avatar.jpg',
+		time: '10:05',
+		text: 'I used this provider to insert a different theme object depending on a person ',
+		emotion: 'Happy',
```

Comment:
Could you try emoji here? In the design graph we have the emoji there, not the text